<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrystalVolumeClass</class>
 <widget class="QMainWindow" name="CrystalVolumeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>CrystalVolume</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#CrystalVolumeClass {
background-color: white;
border: 5px solid #7070AA;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget {
background-color: white;
}

QWidget::hover#centralWidget {
border: 2px solid #7070AA;
}
</string>
   </property>
   <widget class="CrystalTabWidget" name="tabWidgetMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane {
background-color: #FFFFFF;
}

QTabWidget::tab-bar {
left: 2px;
}

QTabBar::scroller {
width: 32px;
}

QToolButton {
background-color: #F0F0F0;
padding: 100px;
border-radius: 6px;
margin: 10px 0px 10px 4px;
}


QToolButton::enabled::hover {
background-color: #606099;
}

QToolButton::!enabled {
background-color: #E0E0E0;
}

QToolButton::enabled {
background-color: #7070AA;
}



QToolButton::right-arrow::selected {
background-color: #7070AA;
padding: 4px;
border-radius: 6px;
}

QTabBar::tab {
height: 30px;
margin: 4px 2px 5px 2px;
border-radius: 5px;
padding-left: 6px;
padding-right: 6px;
}

QTabBar::tab::!selected {
background-color: #F0F0F0;
}

QTabBar::tab::hover {
background-color: #E0E0E0;
}

QTabBar::tab::selected {
color: #FFFFFF;
background-color: #7070AA;
}

QTabBar::close-button::!selected {
margin: 2px;
background-color: #CCCCCC;
border-radius: 6px;
}

QTabBar::close-button::hover {
margin: 2px;
background-color: #AA4444;
border-radius: 6px;
}

QTabBar::close-button::selected {
margin: 2px;
background-color: #FFFFFF;
border-radius: 6px;
}

QTabBar::close-button::selected::hover {
margin: 2px;
background-color: #993333;
border-radius: 6px;
}</string>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddTab">
    <property name="geometry">
     <rect>
      <x>762</x>
      <y>8</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #FFFFFF;
background-color: #7070AA;
border-radius: 6px;
}

QPushButton::hover {
background-color: #606099;
}

QPushButton::pressed {
background-color: #505088;
}


</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>42</y>
      <width>800</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="CrystalCommandLine" name="lineEditCommand">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>46</y>
      <width>400</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {

background-color: #F0F0F0;
border-radius: 6px;
padding-left: 8px;
}

QLineEdit::hover {
background-color: #E0E0E0;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>&gt;_</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CrystalTabWidget</class>
   <extends>QTabWidget</extends>
   <header>CrystalTabWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CrystalCommandLine</class>
   <extends>QLineEdit</extends>
   <header>CrystalCommandLine.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="CrystalVolume.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonAddTab</sender>
   <signal>clicked()</signal>
   <receiver>CrystalVolumeClass</receiver>
   <slot>onClickedPushButtonAddTab()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onClickedPushButtonAddTab()</slot>
 </slots>
</ui>
